servers/display_server.h

```diff
@@ -153,6 +153,8 @@ class DisplayServer : public Object {
 		FEATURE_NATIVE_DIALOG_INPUT,
 		FEATURE_NATIVE_DIALOG_FILE,
 		FEATURE_NATIVE_DIALOG_FILE_EXTRA,
+		FEATURE_NATIVE_DIALOG_FILE_MIME,
+		FEATURE_NATIVE_DIALOG_FILE_EXT,
```

Comment:
The feature flag for file extensions is unnecessary, as it is supported across all platforms (I have already submitted a PR for Android). It would be better to remove this flag, as Android can handle file extensions even without the PR. If the MIME type is not provided, app simply defaults to showing all file types.

Reply:
@bruvzg My PR for adding file extension support for Android is now merged.

Reply:
Removed `FEATURE_NATIVE_DIALOG_FILE_EXT` and updated to use send both extensions and MIMEs to Android Java part.